contrib/kind.sh

```diff
@@ -901,6 +901,9 @@ install_ovn_multiple_nodes_zones() {
   n=1
   for node in $KIND_NODES; do
     zone="zone-${zone_idx}"
+    if [ "${zone}" == "zone-1" ]; then
+      zone="global"
+    fi
```

Comment:
I had the impression this was just a means to reproduce the issue but I was not aware we wanted this change actually in

Reply:
good point..